.circleci/config.yml

```diff
@@ -32,6 +33,43 @@ jobs:
       - run: xcodebuild -version
       - run: xcodebuild test -scheme HoneycombTests -sdk iphonesimulator17.5 -destination 'OS=17.5,name=iPhone 15'
 
+  smoke_tests:
+    macos:
+      xcode: 15.4.0
+    resource_class: macos.m1.medium.gen1
+
+    steps:
+      - attach_workspace:
+          at: ./
+      - macos/preboot-simulator:
+          version: "17.5"
+          platform: "iOS"
+          device: "iPhone 15"
```

Comment:
do we want to use some matrix tests to cover more versions/platforms/devices?

Reply:
if we do test on older iOS versions, we'll have to change the test expectations, because various MetricKit signals were added in different iOS versions. it's probably not trivial

Reply:
I think we should make this a follow up task, but overall yes this sounds like a good idea!